packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1255,11 +1297,11 @@ export class TransactionController extends BaseController<
     return String(txMeta.id);
   }
 
-  private isTransactionCompleted(transactionid: string): {
+  private isTransactionCompleted(transactionID: string): {
```

Comment:
For naming going forward, I think the standard is `transactionId` in code, and `Transaction ID` in labels.